### PR TITLE
[SDK-265] Add pxration signal to  Device Object in ORTB request

### PR DESCRIPTION
**Pull request by ValeriiLoopMe**
As we decided to use Scale in pxration object(ticket SDK-265) I have implemented this and leaved comment in ticket how I tested

### CODE REVIEW

**Comment by bzarzoza** [comment]
Can you show the log from Charles? I'm not able to see if this calculation is accurate with the information provided.

File: framework/LoopMeUnitedSDK/Core/Internal/Builder/LoopMeORTBTools.m [hunk 1]
```diff
@@ -107,6 +107,7 @@ - (NSData *)makeRequestBody {
             @"make": @"Apple",
             @"os": @"iOS",
             @"js": @1,
+            @"pxratio": @([LoopMeIdentityProvider calculatePxratioOfScreen]),
```
Comment (bzarzoza) [inline framework/LoopMeUnitedSDK/Core/Internal/Builder/LoopMeORTBTools.m:110]: This needs to be formatted with `NSNumberFormatter`.

// Assuming `LoopMeIdentityProvider` is already imported and calculatePxratioOfScreen is available

float pxratio = [LoopMeIdentityProvider calculatePxratioOfScreen];
NSNumber *pxratioNumber = @(pxratio);

// Create a number formatter
NSNumberFormatter *numberFormatter = [[NSNumberFormatter alloc] init];
[numberFormatter setNumberStyle:NSNumberFormatterDecimalStyle];
[numberFormatter setMinimumFractionDigits:3];
[numberFormatter setMaximumFractionDigits:3];

// Format the number as a string
NSString *formattedPxratio = [numberFormatter stringFromNumber:pxratioNumber];

// Print the formatted string
NSLog(@"Formatted pxratio: %@", formattedPxratio);

**Review by bzarzoza — CHANGES_REQUESTED** [review]
Formatting should be done with NSNumberFormatter.

File: framework/LoopMeUnitedSDK/Core/Internal/Utility/LoopMeIdentityProvider.m [hunk 2]
```diff
@@ -104,4 +104,14 @@ + (NSNumber *)customAuthorizationStatus {
     }
 }
 
++ (CGFloat)calculatePxratioOfScreen {
+    CGFloat screenWidth = [[UIScreen mainScreen] bounds].size.width;
```
Comment (bzarzoza) [inline framework/LoopMeUnitedSDK/Core/Internal/Utility/LoopMeIdentityProvider.m:108]: I think this calculation is just w/h. This looks like it calculates Pixels per square inch and divides by 1000.

**Review by EpanchinE — APPROVED** [review]

**Pull request merged**